src/app/core/models/common/export-settings.model.ts

```
@@ -87,6 +88,84 @@
+        }
+      ];
+    }
+
+    static dateGrouping(exportType:string, expenseGrouping: string, showApprovedDate: boolean, showVerificationDate: boolean): SelectFormOption[] {
+      
+      const excludedDate = expenseGrouping === ExpenseGroupingFieldOption.EXPENSE_ID
+      ? ExportDateType.LAST_SPENT_AT 
+      : ExportDateType.SPENT_AT;
+      
+      if (exportType === 'CCC') {
+        return this.getCreditCardExpenseGroupingDateOptions().filter(option => option.value !== excludedDate);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)